test/concurrent-array-relationships.js

```diff
@@ -0,0 +1,101 @@
+const t = require('../test-lib/test.js');
+const assert = require('assert');
+const { klona } = require('klona');
+
+describe('Concurrent Array Joins', function() {
+
+  after(async function() {
+    return t.destroy(apos);
+  });
+
+  this.timeout(t.timeout);
+
+  let apos;
+
+  // EXISTENCE
+
+  it('should be a property of the apos object', async function() {
+    apos = await t.create({
```

Comment:
initialization can be done in a `before` block to make tests more independents (see test/widgets.js)